### ISSUE TITLE
failure to autoloot player-owned CELL

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#112]
Report from Discord
> WZ Oblivion Artifacts, 000HatredWell

**Comment by SteveTownsend** [comment]
Autoloot in 000HatredWell worked fine for me. WHitelist, allow player-owned, low global v/w. Got a lot of items, robes off corpse, red glow for locked. Sent feedback to reporting user.

**Comment by SteveTownsend** [comment]
add a message for this case where the Player-owned Cell is not in a LCTN

**Comment by SteveTownsend** [comment]
fixed and tested OK for v1.7.0.0
```
J:\GitHub\SmartHarvestSE\src\WorldState\LocationTracker.cpp(398): [MESSAGE] {24976} (2020-07-09 03:26:01 PM) Player was at location Whiterun, lootable = true, now at unnamed
J:\GitHub\SmartHarvestSE\src\Data\iniSettings.cpp(82): [DEBUG] {24976} (2020-07-09 03:26:01 PM) Get config setting 1/1/notifylocationchange = 1.000000
J:\GitHub\SmartHarvestSE\src\WorldState\LocationTracker.cpp(283): [VERBOSE] {24976} (2020-07-09 03:26:01 PM) GetCellOwner Hit 00000007
J:\GitHub\SmartHarvestSE\src\WorldState\LocationTracker.cpp(485): [DEBUG] {24976} (2020-07-09 03:26:01 PM) Player or Player Faction own cell: no looting
J:\GitHub\SmartHarvestSE\src\WorldState\LocationTracker.cpp(430): [MESSAGE] {24976} (2020-07-09 03:26:01 PM) Player Location unnamed no-loot message
```
